src/components/NavigationItems.js

```diff
@@ -15,6 +15,7 @@ const iconLibrary = {
   'Automate workflows': 'automation',
   'Explore docs': 'developerDocs',
   'Developer champions': 'developerChampions',
+  'Kubecon & CloudNativeCon': 'event',
```

Comment:
It feels a bit odd to have this be a top-level nav item, given that this is a temporary page (and the other top-level nav items are permanent).

Suggest having a top-level "Attend events" nav item with Kubecon nestled underneath.

Reply:
Yes Sir! agreed we need to add that the side nav